surveys/2021.ipynb

```diff
@@ -0,0 +1,1575 @@
G203eujnlPafB7YEJpX2VwQm235J0t/KxcMY4Ael/UEseHHwMC2mQGyfApwCsO6667qDeCIiYjF0lbxvA7pz89dLLNiLHwxg+3FJ21H1jD8NfAD4aNO2Am6zvXMHbc/rbMe2b5S0NfCc7b+WBISkwcBJwGjbD0j6eiOuxqbNTTUtP9dUv6NtBPyv7Z81xyZpFvAR4DpgOtU0xCbAHZLeAzSGxj/eRftn2P5qc/vA5NL+XVQJ9aNUowJflLQT0IjpWOCpTtq/wvaBLOwaYG/gRaoRjvFUyftLtTqDqc5VRET0ka6Gza8EXtU0v7ujpN2b6s0CRkpaTtIGVD01ypDscrYvAL4G7FDqPw0MK6/vAtaStHPZZgVJWy3icXyVBXvc8ErinVvmw5svQg4o+9sNeNL2k03r76CaQ6/br8z5rgmMBW4CLgM+WvaBpPUkvbrUn0iV6CZSJcLDgWmuXGR7ZPmZUuq/VdIaklYC3g1MAv4C7N9os6x/bYv2b6G6OHjB9pO2b6i135hOGCNpozLXfQBwLXA9sKukTUv7K0varNb+EVQ98UeANYHXU13UNWzGwtMLERHRizrtedt26SH+UNJRVEOss6j+Q6+bBNxHdff3TKohYoD1gNMbN0ZRJVmoenAnS3qOqqe4P/BjSauWmH7IggmiU7b/2KLsCUk/LzHNokq0dY9Lug5YhYVHA7B9p6RVJQ2z/XQpvpFq+H9D4Fu2ZwOzy9Dy5NLrfwY4mGo4+RrgaKrkN0/S83Q8Hw1VMj2T6qLh7EZSl3QMcHk5jy9SjWLcX9raAJhoe76kB4A7O2l/MvAdqjnvicBFtl+WNA44R9KrSr1jgL9SzW2/ptSFavTgYdsuca1QYp1CRET0GZX/h5cpkiYAX6r1eDuq9wXgadunlmH3Z2x/r5diGkc1xP+ZXmp/LNUx79NF1UVp8z3ADra/1lm9dddd14cddlhP7bZX5cEkETFQSJpqe3SrdfmGtc79H9XNZtHa8kCrTwFEREQvWiYfCWp7bDfrPU81jI3tr/diSNgeTzWd0FvtTwAm9HCb5/VkexER0T3peUdERLSZZXLOO/rW6NGjPWVK7mmLiFgUmfOOiIhYiiR5R0REtJkk74iIiDazTN5tHn3r8cfv4DfnLfTgthiAPvD+G/s7hIjohvS8IyIi2kySd0RERJtJ8o6IiGgzSd4RERFtJsk7IiKizbRN8pa0pqRp5echSf+oLa/Yje3HSTpxMfe9jqRLFmfbsv0RklauLV8qabUlaO+ZDsoPl/ThTrYbK2mXxd1vi/Y+I+kjPdVeRER0T9t8VMz2o8BIgN5+PGcLRwI/X4LtjwB+BTwLYPsdzRVUPQxctl9e3J3YPrmLKmOpnjd+XXfblLS87Zc6WH0a1bPcT+9uexERseTapufdiqRDJd0k6VZJFzR6t5LeL2lmKZ/YYrt3SposaXhXdYv3AX8q2w6SdHzZ73RJnyjlYyVNkHS+pDslnaXK54B1gaskXVXqzir7HiHpDkknATcDG0j6cq3tb3Ry7MeVmK+X9JpS9nVJXyqvPyfp9tLOuZJGAIcDXyijFW+U9FpJfyl1/iJpw7LteEk/KPEeL+luSWuVdctJ+puk4bafBWZJyoe4IyL6UFsnb+BC2zva3g64A/hYKT8WeFsp37e+gaT3AEcB77A9t7O6pf5GwOO2G8/1/hjwpO0dgR2BQ0sdgO2petlbAhsDu9r+MTAb2MP2Hi2OYXPgl7a3L69fB4yhGmUYJelNLbYZAlxfYp4IHNqizlHA9ra3BQ63PQs4GTjB9kjb1wAnln1vC5wF/Li2/WbAnra/QDVqcFAp3xO4tZw7gCnAG5t3LukwSVMkTXnqqY467hERsTjaPXlvLekaSTOokstWpXwSMF7SocCgWv09gP8E3mn78S7qNqwDPFJb3gv4sKRpwA3AmlQJF+BG2w+Woe9pwIhuHMP9tq+vtb0XcAtVT/z1tbbr/gU05uCndrCf6cBZkg4GOsqeOwNnl9dnArvV1p1ne355fRrQmEv/KAsOkz9MNbKwANun2B5te/Qqq7TN7ExERFto9+Q9HviM7W2AbwCDAWwfDhwDbABMk7RmqX8vMIyqV0kXdRuea7RbCPhs6b2OtL2R7cvLuhdq9ebTvXsK5jW1/b+1tje1/YsW27zoV57l2tF+3gn8FBgFTJXUnVjqz4f9d1y2HwD+KenNwE7AH2v1BlOdo4iI6CPtnryHAXMkrcArw7pI2sT2DbaPBeZSJWaA+4H3Ar+UtFUXdRv+yoI928uAT5Z9ImkzSUO6iPPpEmtXLgM+KmloaXs9Sa/uxnYLkLQcsIHtq4CvAKsBQ1vEcR3wwfL6IODaTpo9lWr4/De1HjlUF0IzFzXGiIhYfO2evL9GNXR9BXBnrfx4STMkzaSaE761scL2XVSJ6jxJm3RWt9SfB9wjadNSdCpwO3Bz2eZndN3DPgX4Y+OGtY6UHvzZwOQyFXA+3Uv6zQYBvypt3EI1z/0EcDHwnsYNa8DngI9Img58CPh8J23+nuoCoPnO8l2BPy9GjBERsZj0yuhrdKTc5DbK9jH9HUt/kTSa6iLgjbWy7YEjbX+os2032WSI//c7W3VWJQaIPFUsYuCQNNX26FbrcidRN9i+qMVc+DJD0lHAJ6lNTRTDqUY/IiKiDyV5d5PtU/s7hv5i+zvAd1qUX9EP4URELPPafc47IiJimZOed/S61VffInOpERE9KD3viIiINpPkHRER0WaSvCMiItpM5ryj193++FNsd/5l/R3GgHTr/m/r7xAiog2l5x0REdFmkrwjIiLaTJJ3REREm0nyjoiIaDNJ3hEREW0mybuNSTpa0m2SppfHfO60iNuPk7Rub8UXERG9Ix8Va1OSdgb2AXaw/YKk4cCKi9jMOGAmMLuHw4uIiF6Unnf7WgeYa/sFANtzgS0kXdSoIOmtki6UNEjSeEkzJc2Q9AVJ+wOjgbNKr30lSaMkXS1pqqTLJK1T2pkg
```

Comment:
Incomplete sentence here.

Reply:
Thanks for catching!
I meant to add that this was the same result as in previous surveys.